### PR TITLE
Delete Functionality

### PR DESCRIPTION
**Pull request by Meggie-H**
The mock data will be replaced in the next pr

### CODE REVIEW

File: src/delete.ts [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+			price: 99,
+			rating: 4.5,
+			numberOfReviews: 1536,
+		},
+	];
+
+	const browser = await chromium.launch();
+	const context = await browser.newContext();
+
+	for (let product of products) {
```
Comment (tanadendere) [inline src/delete.ts:161]: You can use ...`const product`... here since product is not re-assigned inside the body

**Review by tanadendere — APPROVED** [review]
Looks good, can't wait to see it with the real data!

**Review by yanevdw — APPROVED** [review]
Well done! Everything looks good to me.

File: src/delete.ts (hunk 1 above)
Comment (Meggie-H) [reply src/delete.ts:161]: Thank you!

**Pull request merged**